### ISSUE TITLE
CssSyntaxError in plugin 'gulp-autoprefixer'

### ISSUE THREAD

**Issue by keithkml** [Swiip/generator-gulp-angular#402]
This is with a freshly created project. I just ran `yo gulp-angular` and haven't made any modifications.

```
[15:04:06] gulp-inject Nothing to inject into index.scss.
[15:04:07] all files 4.8 kB
[15:04:07] Finished 'scripts' after 869 ms
[15:04:07] gulp-ruby-sass: directory
[15:04:07] gulp-ruby-sass: write index.css
      write index.css.map
[15:04:10] gulp-ruby-sass: write vendor.css
      write vendor.css.map
CssSyntaxError in plugin 'gulp-autoprefixer'
Message:
    /Users/keith/Code/wham/server-editor2/src/editor/index.css.map:3:1: Unknown word
Details:
    reason: Unknown word
    file: /Users/keith/Code/wham/server-editor2/src/editor/index.css.map
    line: 3
    column: 1
    source: {
"version": 3,
"mappings": "AAAA,YAAa;EACX,MAAM,EAAE,OAAO;EACf,UAAU,EAAE,IAAI;EAChB,KAAK,EAAE,IAAI;EACX,OAAO,EAAE,OAAO;;;AAGlB,UAAW;EACT,MAAM,EAAE,KAAK;;AAEb,yBAAe;EACb,KAAK,EAAE,IAAI",
"sources": ["../index.scss"],
"names": [],
"file": "index.css"
}
    fileName: /var/folders/fy/0csn7mgd77v5_8rqw_7l4jdw0000gn/T/gulp-ruby-sass/_14139e58-9ebe-4c0f-beca-73a65bb01ce9/index.css.map
[15:04:10] Finished 'styles' after 4.66 s
```

**Comment by zckrs** [comment]
What version you use ? v0.9.1 or master ?

**Comment by keithkml** [comment]
I just followed the instructions at https://github.com/Swiip/generator-gulp-angular#create-your-project

**Comment by pdrosos** [comment]
I have the same issue and also followed the instructions and installed the generator via npm. I am on Windows 7 with NodeJS 0.12.0. For now I have downgraded gulp-autoprefixer in package.json:
``"gulp-autoprefixer": "~1.0.1"``
but it would be nice if this can be fixed.

**Comment by Swiip** [comment]
Hi, it's fixed in the master: #398. Wait until the 0.10 version which is near now.
